inst/qml/LSTcentralLimitTheorem.qml

```diff
@@ -125,29 +126,43 @@ Form
 		}
 	}
 	
+	
+	function getMinSamplesCLT()
+	{
+		var min = 99999
+		if(cltSampleShowType.currentValue == "first" | cltSampleShowType.currentValue == "last")
+		{
+			min = cltFirstOrLastSamples.value
+		} else
+		{
+			min = cltToSample.value
+		}
+		return min
```

Comment:
```suggestion
		return cltSampleShowType.currentValue === "first" || cltSampleShowType.currentValue === "last" ? cltFirstOrLastSamples.value : cltToSample.value
```
`===` avoids type coercion and just like in R, `||` is prefered because `|` works bitwise (e.g., `1 | 2 === 3`)